MigrationExecutorApp/ChangeFeedProcessortHost.cs

```diff
@@ -79,16 +96,12 @@ public async Task StartAsync()
 
             await this.RunChangeFeedHostAsync();
         }
+
         public async Task CreateCollectionIfNotExistsAsync(string endPointUri, string secretKey, string databaseName, string collectionName, int throughput)
         {
-            using (DocumentClient client = new DocumentClient(new Uri(endPointUri), secretKey))
+            using (CosmosClient client = new CosmosClient(config.DestUri, config.DestSecretKey, new CosmosClientOptions() { AllowBulkExecution = true }))   
```

Comment:
Why are we creating a new client? We already have one for the destination account. Also, clients should not be in `using` statements.

Reply:
Thanks I will resolve.